diff --git a/bindings/Makefile.am b/bindings/Makefile.am
index 3632b3c..cd1df6a 100644
--- a/bindings/Makefile.am
+++ b/bindings/Makefile.am
@@ -69,7 +69,6 @@ python:
           && env \
                ABS_TOP_SRCDIR=${abs_top_srcdir} \
                ABS_TOP_BUILDDIR=${abs_top_builddir} \
-               LD_RUN_PATH=$(libdir) \
                $(PYTHON) ${abs_srcdir}/$@/setup.py build_ext $(PYTHON_RPATH) \
           && env \
                ABS_TOP_SRCDIR=${abs_top_srcdir} \
